benchmarks/validate_report.py

```diff
@@ -63,9 +63,6 @@ def validate_reports(artifacts_dir, report_dir, deviation):
         if not error:
             print(f"Model {model} successfully validated")
 
-    if error:
-        raise Exception("Failures in benchmark validation")
-
```

Comment:
why we remove this?

Reply:
Below are the failures from Benchmark CPU run.

Nothing seems totally wrong. However, we need a good way to differentiate between a one off failure vs something fundamentally broken.

Till we fix that, it is going to raise a github issue everyday. So, removing the raise Exception  will reduce the noise.

Hopefully with window size increasing to 30, we will see a more accurate average value. Once we are confident, then we can turn this back on.

```
Error while validating cpu_percentage_mean for model: eager_mode_vgg16_w4_b8, Expected value: 30.85, Observed value: 50.00
Error while validating TS throughput for model: scripted_mode_bert_w4_b1, Expected value: 17.51, Observed value: 36.03
Error while validating cpu_percentage_mean for model: scripted_mode_bert_w4_b1, Expected value: 34.06, Observed value: 80.00
Error while validating TS throughput for model: scripted_mode_bert_w4_b2, Expected value: 20.16, Observed value: 41.
Error while validating TS throughput for model: scripted_mode_bert_w4_b4, Expected value: 21.29, Observed value: 42.67
Error while validating cpu_percentage_mean for model: scripted_mode_bert_w4_b4, Expected value: 29.45, Observed value: 75.00
Error while validating TS throughput for model: scripted_mode_bert_w4_b8, Expected value: 22.78, Observed value: [42](https://github.com/pytorch/serve/actions/runs/4760803067/jobs/8461463678#step:12:43).44
Error while validating cpu_percentage_mean for model: scripted_mode_bert_w4_b8, Expected value: 27.55, Observed value: 50.00
Error while validating TS throughput for model: scripted_mode_vgg16_w4_b1, Expected value: 13.94, Observed value: 24.31
Error while validating TS throughput for model: scripted_mode_vgg16_w4_b2, Expected value: 16.55, Observed value: 27.25
Error while validating TS throughput for model: scripted_mode_vgg16_w4_b4, Expected value: 17.27, Observed value: 27.13
Error while validating cpu_percentage_mean for model: scripted_mode_vgg16_w4_b4, Expected value: 30.19, Observed value: 55.10
Error while validating TS throughput for model: scripted_mode_vgg16_w4_b8, Expected value: 18.08, Observed value: 26.67
```

Reply:
I see, thanks -- makes sense.

Reply:
I noticed that the throughput failures above are not valid. If throughput is increasing, then it should be a don't care from a validation perspective. Pushed a small change to fix that.